imod/tests/fixtures/flow_transport_simulation_fixture.py

```diff
@@ -196,16 +196,16 @@ def flow_transport_simulation():
         print_option="summary",
         csv_output=False,
         no_ptc=True,
-        outer_dvclose=1.0e-4,
+        outer_dvclose=1.0e-6,
         outer_maximum=500,
         under_relaxation=None,
-        inner_dvclose=1.0e-4,
-        inner_rclose=0.001,
-        inner_maximum=100,
+        inner_dvclose=1.0e-6,
+        inner_rclose=0.0001,
+        inner_maximum=200,
         linear_acceleration="bicgstab",
         scaling_method=None,
         reordering_method=None,
-        relaxation_factor=0.97,
+        relaxation_factor=0.9,
```

Comment:
Why are these changes needed?

Reply:
to achieve a better match between the partitioned model and the unpartitioned model. These force the number of iterations between subdomains to increase, lessening the impact of partitioning on the overall outcome. 
We still get a fairly high inconsistency between the two, and hopefully it will improve once we correctly add settings to the gwtgwt exchange packages.